src/scripts/ops/generate_issues.js

```diff
@@ -0,0 +1,114 @@
+const moment = require('moment')
+const fetch = require('node-fetch')
+const roundRawIssues = require('../../utils/ops/repeatable_tasks.json')
+const {
+    getRoundsSelectQuery,
+    updateRoundRecord
+  } = require('../../airtable/airtable_utils')
+
+const organisation = process.env.GITHUB_ORGANISATION || 'oceanprotocol'
+const repo = process.env.GITHUB_REPOSITORY || 'oceandao'
+const token = process.env.GITHUB_TOKEN
+const buildType = process.eventNames.BUILD_TYPE || 'DEV'
```

Comment:
I have updated this to be `process.env.BUILD_TYPE`

Reply:
Great catch, I missed it